changelogs/fragments/58693-consul_session-validate_certs.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - "consul_session - don't ignore `validate_certs` parameter"
```

```suggestion
  - "consul_session - don't ignore ``validate_certs`` parameter"
```
Changelog ist ReST, not MarkDown.